__mocks__/neon-js.js

```diff
@@ -87,7 +87,10 @@ neonjs.wallet = {
   Account: jest.fn(() => { return { address } }),
   getVerificationScriptFromPublicKey: jest.fn(() => scriptHash),
   isAddress: jest.fn(() => true),
-  isNEP2: jest.fn(() => true)
+  isNEP2: jest.fn(() => true),
+  isWIF: jest.fn(() => false),
+  isPrivateKey: jest.fn(() => false),
+  isPublicKey: jest.fn(() => false)
```

Comment:
I think these global stubs are a bad idea long term and we should work to get rid of them, favorite stubbing in tests where needed.